### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by supertobi**

### CODE REVIEW

File: contrib/de_DE/LC_MESSAGES/calcDistance.po [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Tobias Jakobs <tobias.jakobs@gmail.com>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 1.8.4\n"
+
+#: calcDistance.lua:93
+msgid "Calculate the distance from latitude and longitude in km"
+msgstr "Errechne die Entfernung mit mit Hilfe des Längen- und Breitengrads in km"
```
Comment (houz) [inline contrib/de_DE/LC_MESSAGES/calcDistance.po]: 2x "mit"

File: contrib/de_DE/LC_MESSAGES/gps_select.po [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Tobias Jakobs <tobias.jakobs@gmail.com>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 1.8.4\n"
+
+#: gps_select.lua:59
+msgid "Select all images with GPS information"
+msgstr "Markiert alle Bilder mit GPS Informationen"
```
Comment (houz) [inline contrib/de_DE/LC_MESSAGES/gps_select.po]: eventuell durchkoppeln, also "GPS-Informationen"

File: contrib/de_DE/LC_MESSAGES/gps_select.po [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+#: gps_select.lua:60
+msgid "Select all images without GPS information"
+msgstr "Markiert alle Bilder ohne GPS Informationen"
```
Comment (houz) [inline contrib/de_DE/LC_MESSAGES/gps_select.po]: same as above

File: contrib/de_DE/LC_MESSAGES/hugin.po [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+"Language-Team: \n"
+"X-Generator: Poedit 1.8.4\n"
+
+#: hugin.lua:73
+msgid "hugin not found"
+msgstr "Hugin wurde nicht gefunden"
+
+#: hugin.lua:97
+msgid "Will try to stitch now"
+msgstr "Versuche Bilder zusammen zu fügen"
```
Comment (houz) [inline contrib/de_DE/LC_MESSAGES/hugin.po]: should be "zusammenzufügen", see http://www.duden.de/rechtschreibung/zusammenfuegen

**Comment by supertobi** [comment]
Thanks for the review. Should be fixed now.

**Pull request merged**